homeassistant/components/homekit/__init__.py

```diff
@@ -389,6 +392,8 @@ def start(self, *args):
 
         _LOGGER.debug("Driver start")
         self.hass.add_job(self.driver.start)
+        self.hass.bus.async_fire(EVENT_HOMEKIT_STARTED)
+        async_log_entry(self.hass, "HomeKit", "started", DOMAIN)
```

Comment:
You're calling an async function from a sync context. That's not allowed. Call `log_entry` instead.

Reply:
@balloob oh right, i was just going off your original recommendation. fixed.